utils.js

```diff
@@ -0,0 +1,130 @@
+
+const initialHiddenClassElements = document.getElementsByClassName('initial-hidden');
+const resetButton = document.getElementById('reset-button');
+const startButton = document.getElementById('start-button');
+const userThrowImgDiv = document.getElementById('user-throw-img');
+const opponentThrowImgDiv = document.getElementById('opponent-throw-img');
+const userThrowSpan = document.getElementById('user-throw-span');
+const opponentThrowSpan = document.getElementById('opponent-throw-span');
+const userScoreSpan = document.getElementById('user-score');
+const opponentScoreSpan = document.getElementById('opponent-score');
+const tieScoreSpan = document.getElementById('tie-score');
+const resetSpanCount = document.getElementById('reset-count');
```

Comment:
adding some space between  establishing DOM elements too would also help separate these for final commits.